specs-go/config.go

```diff
@@ -108,6 +108,14 @@ type Platform struct {
 	OS string `json:"os"`
 	// Arch is the architecture
 	Arch string `json:"arch"`
+	// OSVersion specifies the operating system version, for example `10.0.10586`.
```

Comment:
I'd pick a more up to date build. And it's actually a quad-tuple these days (it was a triplet). eg `10.0.14393.1198` is the current fully-patched Windows Server 2016 RS1 build number.